### PR TITLE
Dump junit.functest.xml at tier1 tests

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

**Is this a BUG FIX or a FEATURE ?**:
/kind enhancement

**What this PR does / why we need it**:
tier1 test were not dumping junit rests, this PR activate them so they can be seen at jenkins.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/openshift.cnv.workflow.sh [hunk 1]
```diff
@@ -24,4 +24,4 @@ fi
 
 # Run workflow tests
 focus='test_id:3796|test_id:3795|test_id:3813|test_id:3794|test_id:3793'
-make test/e2e E2E_TEST_TIMEOUT=${TIMEOUT} E2E_TEST_ARGS="-ginkgo.noColor -ginkgo.focus $focus" NAMESPACE=$NAMESPACE KUBECONFIG=$KUBECONFIG
+make test/e2e E2E_TEST_TIMEOUT=${TIMEOUT} E2E_TEST_ARGS="--junit-output=junit.functest.xml -ginkgo.noColor -ginkgo.focus $focus" NAMESPACE=$NAMESPACE KUBECONFIG=$KUBECONFIG
```
Comment (phoracek) [inline test/e2e/openshift.cnv.workflow.sh:27]: You prepared a nice explanation in the PR description, please use it in commit message.

**Comment by qinqon** [comment]
/override pull-kubernetes-nmstate-e2e-k8s

We are just chaging openshift workflow script

**Comment by qinqon** [comment]
/override pull-kubernetes-nmstate-e2e-k8s

Not related, we may need some backporting from master

**Pull request merged**